_posts/blog/2023-06-20-ghost-buses.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "Breakout Group Highlight: Ghost Buses"
+description: "Chi Hack Night is excited to be partnering with Hacks/Hackers for Media Party Chicago 2023! Join us as attendees and participants along with journalists, designers, entrepreneurs, and tech experts, sparking innovative dialogues on artificial intelligence, democracy, and the future of media!"
+date: 2023-06-20
+image: /images/blog/ghost-buses.png
+author: Shriya Jalukar
+author_image: /images/people/placeholder-headshot.jpg
+author_bio: "Chi Hack Night Board Member"
+author2:
+author2_image:
+author2_bio: 
+published: true
+featured: true
+redirect_from: /blog/2023/06/20/ghost-buses.html
+---
+<p class="text-center"><img src="/images/blog/ghost-buses.png" alt="Learn more about the Ghost Buses breakout group at Chi Hack Night" class='img-responsive' style='width:100%;'/>
+</p>
+
+Q. "What is a ghost bus?"
+
+A. A "ghost bus" is when a bus tracker app claims that a bus is coming but it never shows up. If you live in Chicago and regularly take public transit, you may have been "ghosted" by a bus!
+
+Q. "What's the goal of this project?"
+
+A. The goal of this project has been to "use public data to assess the discrepancy between scheduled and actual service levels, to understand the prevalence and geographic distribution of "ghost buses"."
+
+Q. "Why did the Ghost Buses group start?" 
+
+A. Good question! The Ghost Buses group's story began in Spring 2022, when Chi Hack Night members started to see more reports of ghost buses on Twitter and in the news --- in particular Streetsblog published an [article](https://chi.streetsblog.org/2022/04/25/data-analysis-found-cta-has-only-been-running-about-half-its-schedule-blue-line-runs/) that only half the Blue Line trains were running. This article inspired CHN member Laurie Merrell to try to do an analogous data analysis for CTA buses. Many CHN breakout groups start when CHN members engage with issues of public concern and then feel inspired to start breakout groups.
+
+Q. "How did the Ghost Buses group get up and running?" 
+
+Anyone can start a breakout group --- here is a [guide to starting and running breakout groups](https://chihacknight.org/blog/2020/02/28/so-you-want-to-start-a-breakout-group.html) at CHN. In our case, Laurie was the CHN member who got the Ghost Buses breakout group started --- she opened up an issue under the CHN GitHub repository, writing up a [short description of the group](https://github.com/chihacknight/breakout-groups/issues/217) to help orient people who were new to the breakout group, announcing the group at the weekly CHN meetings, and creating the Slack channel #ghost-buses-breakout. Gradually, we introduced a google doc for meeting notes, a project board on GitHub, READMEs, and contribution protocols. Since the beginning, many new members have joined and contributed to the project.
+
+(Picture of group meeting)
```

Comment:
I looked at the video that I thought had footage of us meeting and I don't think there are any super great stills, so maybe we can just replace this with a screenshot of the site itself (the map page) somewhere in the post?